src/desktop.md

```diff
@@ -357,14 +357,36 @@ impact your distributable application.
 
 #### Linux
 
+The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 
+executable binary in the `bundle` directory there are two directories, `lib` which contains required `.so`
+library files, and the `data` directory that contains data assets that your application relies on.
```

Comment:
Nit: s/that/which/ for parallel structure.

Reply:
Re-worded, but I'm unsure if I like the new structure. I'll let @sfshaza2 word smith my prose =)